tests/page-objects/apdex/page.js

```diff
@@ -155,17 +157,20 @@ module.exports = class Page {
 
     await this.clickElement(FORM_SUBMIT_SELECTOR);
     await this.assertMany(form.postSubmitAsserts);
-    await this.navigate(form.postTestPath);
+    if (form.useFAB) {
+      await this.navigate(form.postTestPath);
+    }
```

Comment:
I think this shouldn't be depending on FAB. if a form doesn't have a postTestPath, just don't execute it. 
```suggestion
   await this.navigate(form.postTestPath);
```

Reply:
Done thanks